docs/src/tty.rst

```diff
@@ -97,4 +114,12 @@ API
 
     Gets the current Window size. On success it returns 0.
 
+.. c:function:: int uv_tty_set_vterm_state(uv_tty_t* handle, uv_vtermstate_t state)
+
+    Set the Console to use the specified virtual terminal mode.
+
+    This function is only implemented on Windows.
+    .. versionadded:: 1.25.0
```

Comment:
```suggestion

    .. versionadded:: 1.25.0
```

I don't know if the extra space is required for the markdown format, but the other places in the file seem to have it.